### ISSUE TITLE
Our history: HTML5 validation error

### ISSUE THREAD

**Issue by sargentfrancesca** [IATI/preview-website#182]
http://staging2.iatistandard.org/en/about/iati-history/

```
Error: End tag main seen, but there were open elements.

From line 459, column 9; to line 459, column 15

↩↩        </main>↩↩    

Error: Unclosed element div.

From line 150, column 1; to line 150, column 17

 </div>↩↩↩<div class="row">↩    <div
```

see: https://html5.validator.nu/?doc=http%3A%2F%2Fstaging2.iatistandard.org%2Fen%2Fabout%2Fiati-history%2F&showimagereport=yes&showsource=yes Error 11

**Comment by benjamincoleman** [comment]
Looks like an implementation error

**Comment by benjamincoleman** [comment]
Yep, this isn't for the pattern library - it's an implementation issue. 

https://html5.validator.nu/?doc=https%3A//iatistandard.org/en/about/iati-history/&showimagereport=yes&showsource=yes#l459c15

`<div class="row">` isn't closed in this template: https://github.com/IATI/preview-website/blob/master/iati/about/templates/about/history_page.html

I've removed the "for pattern library tag

**Comment by Ocre42** [comment]
The issue appears to occur whenever there's an odd number of history blocks, as the if statement in the template doesn't insert the closing div since it doesn't know which the last template-row is. It's not something that breaks the site so we'll de-prioritise this until we have more time to look into it.
